upgrade/util/util_test.go

```diff
@@ -46,3 +63,108 @@ func TestProgressMonitorInc(t *testing.T) {
 		t.Fatalf(`targetValue = %v, expectedTargetValue = %v`, targetValue, expectedTargetValue)
 	}
 }
+
+func TestCheckUpgradePathSupported(t *testing.T) {
```

Comment:
This is good.

Reply:
Added more test cases for any pre-GA version upgrade is not allowed.

Reply:
Removed the non-GA version test cases.